blender/arm/log.py

```diff
@@ -2,8 +2,8 @@
 
 DEBUG = 36
 INFO = 37
-WARN = 35
-ERROR = 31
+WARN = 93
```

Comment:
I'm not sure if it makes sense to use light yellow for warnings (at least that's how it looks like on Windows). If there are large blocks of white text in the console, a yellow warning in between might get overlooked.